### ISSUE TITLE
The correct way to check if a character is grounded

### ISSUE THREAD

**Issue by maxwellodri** [idanarye/bevy-tnua#4]
I've been checking whether the jump_velocity in the TnuaPlatformerAnimatingOutput is Some but not 100% sure if this correct.

**Comment by diego411** [comment]
@idanarye Would the new way to do this be to check whether get_count_mut of TnuaSimpleAirActionsCounter is Some?

**Comment by idanarye** [comment]
@diego411 No. The new way is to use [`controller.is_airborne`](https://docs.rs/bevy-tnua/latest/bevy_tnua/controller/struct.TnuaController.html#method.is_airborne). The air actions counter will probably give you the same information, but:

* It's a hassle to set up (especially if you don't already use it for air actions)
* It will make the intentions of the code less clear.
* If you want to switch to a different air counter, you'll have to change the code that checks for groundedness.

**Comment by diego411** [comment]
@idanarye Thank you! Idk how i missed that
